airbyte-integrations/connectors/destination-postgres/build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.44.3'
+    cdkVersionRequired = '0.46.0'
     features = ['db-destinations', 'datastore-postgres', 'typing-deduping']
     useLocalCdk = false
```

Comment:
nit: why not local cdk here as well? (or conversely, why not upgrade to 0.46.0 on strict-encrypt)

Reply:
because I'm lazy and forgot strict-encrypt